js/app/foodtech/dashboard/redux/middlewares.js

```diff
@@ -143,6 +148,19 @@ export const notification = ({ getState }) => {
       }
     }
 
+    if (action.type === ORDER_DELAYED) {
```

Comment:
Would be better at some point to generalize this behavior server-side, by marking PHP events with something, so that the fronted knows which event should trigger this type of notification. But it's ok for now 🙂